tox.ini

```diff
@@ -12,6 +12,7 @@ commands =
 	pytest {posargs}
 passenv =
 	HOME
+	with-ssh-agent: SSH_AUTH_SOCK
```

Comment:
I'm guessing you had to add this because your checkout is with SSH and relies on that variable to clone the upstream repo again?